### PR TITLE
Add compass over map

### PR DESCRIPTION
**Pull request by sylvaindnd**
# Feature

## Interractive map
- Add a compass over map

### CODE REVIEW

**Review by Lucas-Malatchoumy — DISMISSED** [review]

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -51,5 +51,10 @@
 	<string>This app needs camera access to scan QR codes</string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>This app needs photos access to get QR code from photo library</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>This app needs your location to provide relevant features and information when you are using it.<string>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>This app needs your location to provide relevant features and information both when you are using it and in the background.</string>
+
```
Comment (FabienHP) [inline ios/Runner/Info.plist]:
```suggestion

```
Not necessarie

File: android/app/build.gradle [hunk 2]
```diff
@@ -48,7 +48,7 @@ android {
         applicationId "com.example.front_end"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion 21
+        minSdkVersion 23
```
Comment (FabienHP) [inline android/app/build.gradle:51]:
```suggestion
        minSdkVersion 21
```
Dont need to upgrade sdkVersion

File: .vscode/settings.json
Comment (FabienHP) [inline .vscode/settings.json:1]: Delete this file, its personnal vscode setting

File: .vscode/launch.json
Comment (FabienHP) [inline .vscode/launch.json:1]: Delete this file, its personnal vscode setting

File: assets/compass.png
Comment (FabienHP) [inline assets/compass.png:1]: Move this image in assets/images/

File: lib/screens/home_page.dart [hunk 3]
```diff
@@ -25,7 +25,7 @@ class _HomePageState extends State<HomePage> {
     return MaterialApp(
       home: Scaffold(
         appBar: AppBar(
-          title: const Text('Toto'),
+          title: const Text('Vivatech'),
```
Comment (FabienHP) [inline lib/screens/home_page.dart:28]: Possible to add official logo of Vivatech ?

**Review by FabienHP — CHANGES_REQUESTED** [review]

File: ios/Runner/Info.plist (hunk 1 above)
Comment (sylvaindnd) [reply ios/Runner/Info.plist]: _**NSLocationWhenInUseUsageDescription**_ and _**NSLocationAlwaysAndWhenInUseUsageDescription**_ are the users permissions for IOS. They allows access to the location setting of the Iphone to have access to the compass.

I just saw that I forgot to close the _**<string>**_ tag on the line **55**, I will correct that.

File: android/app/build.gradle (hunk 2 above)
Comment (sylvaindnd) [reply android/app/build.gradle:51]: The minimum version of the SDK to run the compass is 23.

File: lib/screens/home_page.dart (hunk 3 above)
Comment (sylvaindnd) [reply lib/screens/home_page.dart:28]: Not possible in the _**title**_ parameter of the MaterialApp widget

**Review by FabienHP — APPROVED** [review]

**Review by Lucas-Malatchoumy — APPROVED** [review]

**Pull request merged**